app/src/pages/settings/AnnotationConfigDialog.tsx

```diff
@@ -0,0 +1,424 @@
+                  name="annotationType"
+                  render={({ field }) => (
+                    <RadioGroup
+                      {...field}
+                      aria-label="Type"
+                      data-testid="type-picker"
+                      direction="column"
+                      isReadOnly={mode === "edit"}
+                    >
+                      <Label>Type</Label>
+                      {types
+                        .filter((type) =>
+                          mode === "edit" ? type === field.value : type
+                        )
+                        .map((type) => (
+                          <Radio key={type} value={type}>
+                            {type.charAt(0).toUpperCase() +
+                              type.slice(1).toLowerCase()}
+                          </Radio>
+                        ))}
+                    </RadioGroup>
+                  )}
+                />
+                {(annotationType === "CONTINUOUS" ||
+                  annotationType === "CATEGORICAL") && (
+                  <Controller
+                    control={control}
+                    name="optimizationDirection"
+                    render={({ field }) => (
+                      <RadioGroup
+                        {...field}
+                        aria-label="Optimization Direction"
+                        data-testid="optimization-direction-picker"
+                        direction="column"
+                      >
+                        <Label>Optimization Direction</Label>
+                        {optimizationDirections.map((direction) => (
+                          <Radio key={direction} value={direction}>
+                            {direction.charAt(0).toUpperCase() +
+                              direction.slice(1).toLowerCase()}
+                          </Radio>
+                        ))}
+                      </RadioGroup>
+                    )}
+                  />
+                )}
+                {annotationType === "CONTINUOUS" && (
+                  <>
+                    <Controller
+                      control={control}
+                      name="lowerBound"
+                      rules={{
+                        required: "Min is required",
+                      }}
+                      render={({
+                        field: { value, ...field },
+                        fieldState: { error },
+                      }) => (
+                        <NumberField
+                          {...field}
+                          value={typeof value === "number" ? value : undefined}
+                          isInvalid={!!error}
+                        >
+                          <Label>Min</Label>
+                          <Input placeholder="0" />
+                          <FieldError>{error?.message}</FieldError>
+                        </NumberField>
+                      )}
+                    />
+                    <Controller
```

Comment:
can you put the min and max on a single line and somehow describe that it's a range?
<img width="873" alt="Screenshot 2025-03-24 at 9 29 24 AM" src="https://github.com/user-attachments/assets/8c57868a-6f2c-4df2-95d8-d3f6682127ff" />